test/quantization/test_quantized_op.py

```diff
@@ -2395,6 +2395,100 @@ def test_custom_module_lstm(self):
                         msg=(f"Error is too high: SNR(dB): {power}, "
                              f"Signal: {signal}, MSE: {mse}"))
 
+    @override_qengines
+    def test_custom_module_multi_head_attention(self):
```

Comment:
Can we also add a test for QAT, i.e ensuring that we can create a model with fake quant after prepare_qat, followed by convert?

Reply:
Prepare_qat is not implemented -- will send it as a separate PR